streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -2338,7 +2338,6 @@ private TopologyDescription.Sink addSink(final String sinkName,
         return expectedSinkNode;
     }
 
-    @Deprecated // testing old PAPI
```

Comment:
Code was already updated for new PAPI. Tag and comment are just outdated, so removing it.